build-tools/packages/build-cli/src/commands/release/report.ts

```diff
@@ -567,11 +576,19 @@ export default class ReleaseReportCommand extends ReleaseReportBaseCommand<
 
 			const isNewRelease = this.isRecentReleaseByDate(latestDate);
 			const scheme = detectVersionScheme(latestVer);
-			const ranges = getRanges(latestVer);
+
+			if (context.flubConfig.releaseReport === undefined) {
+				throw new Error(`legacy-compat release report not found.`);
```

Comment:
I think the error would be better phrased saying `releaseReport` was not found in config.